styles.css

```diff
@@ -843,9 +887,9 @@ main {
     width: 1.6rem;
   }
 
-  #sec2 {
+  /* #sec2 {
     padding: 10rem, 0;
-  }
+  } */
```

Comment:
- [ ] Kindly avoid leaving comments in your code. It makes your code unclean :+1:  You can check out this [thread](https://questions.microverse.org/t/is-writing-down-comments-on-the-code-a-good-practice/687) to read more about this :+1: